docs/recipes/quickly-switch between-multiple-foam-repos.md

```diff
@@ -0,0 +1,32 @@
+# Quickly switch between multiple Foam repos
+
+this #recipe is allow to Quickly switch between multiple Foam repos. and sorry for Setting/Content's Hangul the Korean Character.
+
+## Installation
+
+**This extension is not included in the template**
+
+This Method need [Project Manager](https://marketplace.visualstudio.com/items?itemName=alefragnani.project-manager) extension, so please install this extension.
+
+## Quickly switch between multiple Foam repos - Git Folder
+First, Need arrange Quickly switching Multiful Foam repos in one folder
+
+<img width="1322" alt="스크린샷 2021-06-02 오전 10 05 29" src="https://user-images.githubusercontent.com/1904967/120408485-1af37c80-c38a-11eb-8454-713a63ba5c7b.png">
+Second, In VSCode's Extensnion setting, select arrange folder, and extension setting, assign to this folder.
+```Project Manager › Git: Base Folders```
```

Comment:
There's something wrong with your use of backtics. This does not render correctly in github.
You can see your version here:
https://github.com/readingsnail/foam/blob/master/docs/recipes/quickly-switch%20between-multiple-foam-repos.md

You could probably write:

```
Project Manager › Git: Base Folders
Indicates the base folders to search for Git projects
```

Though I am not sure what you are trying to achieve with a code block in this part of the text?